### DISCUSSION TITLE
What's the process to get mods to work on Xbox Gamepass for PC (ex: Palworld)?

### DISCUSSION THREAD

**Discussion by JRDK92** [UE4SS-RE/RE-UE4SS#390] [General]
I installed a couple of mods off of Nexus mods for Palworld into my mods folder:
C:\XboxGames\Palworld\Content\Engine\Binaries\WinGDK\Mods

![image](https://github.com/UE4SS-RE/RE-UE4SS/assets/32148757/cf93819e-02c5-4ff5-9867-04328bb58798)

Then, I launched the game. The mods did not appear in game at all. My carry capacity should've increased, and food timer gone up. I opened mods.txt, and added the mods there 1 by 1, and saved it... But it seemed to do nothing? 

CheatManagerEnablerMod : 1
ActorDumperMod : 0
ConsoleCommandsMod : 1
ConsoleEnablerMod : 1
SplitScreenMod : 0
LineTraceMod : 0
BPModLoaderMod : 1
BPML_GenericFunctions : 1
jsbLuaProfilerMod : 0

On my Laptop computer, following these steps worked perfectly, 100% using this exact set of steps. On my desktop PC, however, no mods will appear in game for some reason....

**Comment by UE4SS** [comment]
UE4SS and its mods don't usually go in the `Engine\Binaries\...` directory, they usually go in the `<GameName>\Binaries\...` directory.
Could that be the problem ?

**Reply by JRDK92** [reply]
That was definitely the issue.... Thanks 👍
